.changeset/twenty-lamps-decide.md

```diff
@@ -0,0 +1,8 @@
+---
+'@srcbook/api': patch
+'@srcbook/shared': patch
+'@srcbook/web': patch
+'srcbook': patch
+---
+
+Adds autocomplete support!
```

Comment:
Since this isn't part of the commit, can we add the shortened git sha linking to the actual commit?